.circleci/config.yml

```diff
@@ -12,9 +12,10 @@ executors:
   ruby_browsers:
     docker:
       # Specify the Ruby version you desire here
-      - image: circleci/ruby:2.6.6-node-browsers
+      - image: circleci/ruby:2.7.3-node-browsers
         environment:
           RAILS_ENV: test
+          BUNDLER_VERSION: 1.17.3
```

Comment:
😭 🤠

Reply:
I...don't know what the plan is about that to be honest 😬